client/data/users/use-users-query.js

```diff
@@ -38,10 +38,19 @@ const useUsersQuery = ( siteId, fetchOptions = {}, queryOptions = {} ) => {
 			 * such as Administrator and Editor, and has also been added as a "Viewer" .
 			 */
 			const users = uniqueBy( extractPages( data.pages ), compareUnique );
+			const found = users?.length ?? data.pages[ 0 ].found;
+			const totalUsers = data.pages?.[ 0 ]?.total_users || found;
 			return {
-				users: uniqueBy( extractPages( data.pages ), compareUnique ),
-				total: users?.length ?? data.pages[ 0 ].found,
 				...data,
+				users,
+				/*
+				 * @TODO: The property name `total` is misleading.
+				 * The value `found` is the total number of users returned from the query,
+				 * not the total number of users on the site. It should be renamed to `found`,
+				 * and the `totalUsers` property should be renamed to `total`.
+				 */
```

Comment:
I think this should be done in another found. `total` is inaccurate and misleading.

Reply:
Indeed. `found` sounds more clear